SAMtemplates/state_machines/main.yaml

```diff
@@ -53,6 +75,9 @@ Resources:
         AdditionalPolicies: !Join
           - ","
           - - Fn::ImportValue: !Sub ${StackName}:functions:${GetMyPrescriptionsFunctionName}:ExecuteLambdaPolicyArn
+            - Fn::ImportValue: !Sub ${StackName}:functions:${EnrichPrescriptionsFunctionName}:ExecuteLambdaPolicyArn
+            - !ImportValue psu:functions:psu-GetStatusUpdates:ExecuteLambdaPolicyArn
+            - !Ref ToggleGetStatusUpdatesParameterManagedPolicyArn
```

Comment:
in fact instead of !Ref, could we export the policy ARN in paramaters/main.yaml instead of just creating an output and then we dont need to pass it around - we can just use Fn::ImportValue

Reply:
Have exported the arn of the managed policy, as suggested.